tools/benchmark_tool/openvino/tools/benchmark/utils/statistics_report.py

```diff
@@ -5,7 +5,7 @@
 import os
 import sys
 from enum import Enum
-
+import csv
```

Comment:
Do we need to add this module to the requirements file?

Reply:
The third path "csv" wheel is useful in data analysis
If possible, pls add it in requirement file